ieee802154 tests fail to build

tests/net/ieee802154/fragment/net.ieee802154.fragment & tests/net/ieee802154/l2/net.ieee802154.l2 do not build in sanitycheck due to inclusion of "ieee802154_fragment.h":

/home/buildslave/src/github.com/zephyrproject-rtos/ci-test/zephyr/tests/net/ieee802154/fragment/src/main.c:27:33: fatal error: ieee802154_fragment.h: No such file or directory
 #include "ieee802154_fragment.h"

--- a/tests/net/ieee802154/crypto/CMakeLists.txt
+++ b/tests/net/ieee802154/crypto/CMakeLists.txt
@@ -1,6 +1,11 @@
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(NONE)
 
-target_include_directories(app PRIVATE $ENV{ZEPHYR_BASE}/subsys/net/ip $ENV{ZEPHYR_BASE}/subsys/net/ip/l2/ieee802154)
+target_include_directories(
+  app
+  PRIVATE
+  $ENV{ZEPHYR_BASE}/subsys/net/ip
+  $ENV{ZEPHYR_BASE}/subsys/net/l2/ieee802154
+  )
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})

--- a/tests/net/ieee802154/fragment/CMakeLists.txt
+++ b/tests/net/ieee802154/fragment/CMakeLists.txt
@@ -1,6 +1,11 @@
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(NONE)
 
-target_include_directories(app PRIVATE $ENV{ZEPHYR_BASE}/subsys/net/ip $ENV{ZEPHYR_BASE}/subsys/net/ip/l2/ieee802154)
+target_include_directories(
+  app
+  PRIVATE
+  $ENV{ZEPHYR_BASE}/subsys/net/ip
+  $ENV{ZEPHYR_BASE}/subsys/net/l2/ieee802154
+  )
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})

--- a/tests/net/ieee802154/l2/CMakeLists.txt
+++ b/tests/net/ieee802154/l2/CMakeLists.txt
@@ -1,6 +1,11 @@
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(NONE)
 
-target_include_directories(app PRIVATE $ENV{ZEPHYR_BASE}/subsys/net/ip $ENV{ZEPHYR_BASE}/subsys/net/ip/l2/ieee802154)
+target_include_directories(
+  app
+  PRIVATE
+  $ENV{ZEPHYR_BASE}/subsys/net/ip
+  $ENV{ZEPHYR_BASE}/subsys/net/l2/ieee802154
+  )
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})

